src/v/pandaproxy/schema_registry/requests/get_schemas_ids_id.h

```diff
@@ -24,6 +24,10 @@ inline void rjson_serialize(
   ::json::Writer<::json::StringBuffer>& w,
   const get_schemas_ids_id_response& res) {
     w.StartObject();
+    if (res.definition.type() != schema_type::avro) {
```

Comment:
Should we always return `schemaType` regardless of whether its `AVRO`, `PROTOBUF`, or `JSON`?

Reply:
This duplicates the behaviour of the "other" implementation.

Reply:
Specifically https://github.com/confluentinc/schema-registry/pull/1288 it appears to be related to backwards compat.